### ISSUE TITLE
Cannot open file edited on Android in Windows

### ISSUE THREAD

**Issue by jmacura** [OpenOrienteering/mapper#1115]
### Steps to reproduce
1. Transfer .omap file from Win to Android device (tablet)
2. Go to forest, have a nice 2 hour-long walk with GPS on and some casual editing in OOMapper
3. Transfer edited .omap file back to Windows PC
4. Try to open file in OOMapper

### Actual behaviour
Opening of the file immediately crashes with:
```
 Cannot open file C:\...\...\polom_honza_180712.omap

 Could not read 'C:\...\...\polom_honza_180712.omap':
```

### Expected behaviour
File should open as usual. This process always worked for me so far (some 20 times before). This is the first time it happens.
I'd like to provide more info, but don't know where to find it. The file is not corrupted as far as I can tell -- it opens on Android version correctly (except for some missing templates which are really missing), the transfer was complete, it has the same size, I've double-checked it...

### Configuration
Mapper Version: 0.8.1
Operating System: Windows 10 Pro (v. 1803)/Android 7.0

**Comment by ollesmaps** [comment]
The transfer might somehow corrupt it. You can try email, cloud transfer from Android device.

**Comment by jmacura** [comment]
I have triple-checked it and the files in tablet and in the PC are character-wise equal.

As I emailed to @dg0yt , the file really **is** corrupted, but in tablet it opens without any problem, which I absolutely don't understand...

**Comment by jmacura** [comment]
I have now came up with this idea: open the file in tablet, make several random edits, just to change the file, save it, transfer it, open it in PC.
Didn't work :-(
The file ending is different (the last XML element is not broken in the middle), but opening it in the Win version still ends with the very same error...

**Comment by jmacura** [comment]
So I have found how to fix the problem, yet I have no idea how and why it happens in the first place..

**Story**
I had also other unpleasant problems with the same Android tablet: the GPX files created by OOMapper were completely invisible outside of the Android device (didn't appeared in Windows Explorer when connected, but showed up in Android File manager). Then I have realized that _all_ the files created by some Android app are invisible from outside. So I did some googling and I guess it's some issue related either to these: https://stackoverflow.com/questions/7391432/unable-to-see-file-in-windows-explorer-while-it-is-visible-in-android-file-brows and https://stackoverflow.com/questions/7429087/cant-see-a-file-in-windows-written-by-an-android-app-on-sd-card-unless-i-force (i.e. some not-flushed buffer) or to this: https://android.stackexchange.com/questions/165818/some-folders-and-files-not-showing-up-on-pc-when-connected-via-usb Anyway, it all was "fixed" by restarting the tablet. The .omap file transferred afterwards is now opening correctly on Windows PC and .GPX files are visible.  I can only hope that I will not have to restart it everytime I will want to transfer the files from Android to PC...

**TL;DR**
if it happens, restart the Android device.

**Comment by lpechacek** [comment]
It looks to me like this issue is resolved. Correct me if I'm wrong.

**Comment by dg0yt** [comment]
The issue is resolved, but it might be worth having a remark in the Android or FAQ section of the manual. I feel like this is repeating.

**Comment by dg0yt** [comment]
I just noticed that our Android storage manual page already recommends a reboot if one cannot find a recorded GPX track... So spinning this further, I need to touch:
- GPX tracks
- scribble images.
